api/query/search.go

```diff
@@ -46,9 +47,24 @@ type SearchResult struct {
 	LegacyStatus []LegacySearchRunResult `json:"legacy_status"`
 
 	// Subtests (names) which are included in the LegacyStatus summary.
+	Subtests mapset.Set `json:"-"`
+}
+
+type searchResultNoCustomMarshalling SearchResult
+type marshallableSearchResult struct {
+	searchResultNoCustomMarshalling
 	Subtests []string `json:"subtests,omitempty"`
```

Comment:
Subtests []string `json:"subtests,omitempty" - just a quick question about this syntax - is Subtests a variable being declared here? if so what is `json:"subtests,omitempty" in the end?

Reply:
https://gobyexample.com/json

On the struct, `Subtests` is the field name. the `json:"field_name"` tag renames the field's name when serializing (marshalling) as JSON. `,omitempty` is a helper that drops the field when its value is empty.